translate/api/TranslateV3Samples/Program.cs

```diff
@@ -0,0 +1,337 @@
+
+        [Option("project_id", Default = "[Google Cloud Project ID]")]
+        public string ProjectId { get; set; }
+    }
+
+    [Verb("getGlossary", HelpText = "")]
+    internal class GetGlossaryOptions
+    {
+        [Option("project_id", Default = "[Google Cloud Project ID]")]
+        public string ProjectId { get; set; }
+
+        [Option("glossary_id", Default = "[Glossary ID]")]
+        public string GlossaryId { get; set; }
+    }
+
+    [Verb("getSupportedLanguages", HelpText = "")]
+    internal class GetSupportedLanguagesOptions
+    {
+        [Option("project_id", Default = "[Google Cloud Project ID]")]
+        public string ProjectId { get; set; }
+    }
+    [Verb("getSupportedLanguagesWithTarget", HelpText = "")]
+    internal class GetSupportedLanguagesWithTargetOptions
+    {
+        [Option("language_code", Default = "en")]
+        public string LanguageCode { get; set; }
+
+        [Option("project_id", Default = "[Google Cloud Project ID]")]
+        public string ProjectId { get; set; }
+    }
+
+    [Verb("listGlossaries", HelpText = "")]
+    internal class ListGlossaryOptions
+    {
+        [Option("project_id", Default = "[Google Cloud Project ID]")]
+        public string ProjectId { get; set; }
+    }
+
+    [Verb("translateText", HelpText = "")]
+    internal class TranslateTextOptions : TranslateTextBaseOptions { }
+
+    [Verb("translateTextWithGlossary", HelpText = "")]
+    internal class TranslateTextWithGlossaryOptions : TranslateTextBaseOptions
+    {
+        [Option("source_language", Default = "en")]
+        public string SourceLanguage { get; set; }
+
+        [Option("glossary_id", Default = "[YOUR_GLOSSARY_ID]")]
+        public string GlossaryId { get; set; }
+    }
+
+    [Verb("translateTextWithModel", HelpText = "")]
+    internal class TranslateTextWithModelOptions : TranslateTextBaseOptions
+    {
+        [Option("model_id", Default = "[MODEL ID]")]
+        public string ModelId { get; set; }
+
+        [Option("source_language", Default = "en")]
+        public string SourceLanguage { get; set; }
+
+        [Option("location", Default = "global")]
+        public string Location { get; set; }
+    }
+
+    [Verb("translateTextWithGlossaryAndModel", HelpText = "")]
+    internal class TranslateTextWithGlossaryAndModelOptions : TranslateTextBaseOptions
+    {
+        [Option("model_id", Default = "[MODEL ID]")]
+        public string ModelId { get; set; }
+
+        [Option("source_language", Default = "en")]
+        public string SourceLanguage { get; set; }
+
+        [Option("location", Default = "global")]
+        public string Location { get; set; }
+
+        [Option("glossary_id", Default = "[YOUR_GLOSSARY_ID]")]
+        public string GlossaryId { get; set; }
+    }
+
+    public class TranslateV3Samples
+    {
+        public static void Main(string[] args)
+        {
+            // Command line argument parser we use doesn't support more than 15 options, therefore
```

Comment:
Should this comment be just before the switch and not here? And should it say "verbs" instead of options?